src/pages/Game.tsx

```diff
@@ -411,6 +448,43 @@ const GamePage = ({ ws, currentUser }: GameProps): JSX.Element => {
     mjGameState.requestRedraw();
   };
 
+  /**
+   * For WINNING_TILES
+   * @param payload WinningTilesJSON
+   */
+  const mjWinningTiles = (payload: unknown): void => {
+    const data = payload as WinningTilesJSON;
+    const { connectionId, handPointResults } = data;
+    const mjGameState = gameState as MahjongGameState;
+    mjGameState.endRound();
+    mjGameState.winnerFound();
+    mjGameState.setWinnerInfo(connectionId, handPointResults);
+    mjGameState.requestRedraw();
+  };
+
+  /**
+   * For UPDATE_GAME_STATE
+   * @param payload UpdateGameStateJSON
+   */
+  const mjUpdateGameState = (payload: unknown): void => {
+    const data = payload as UpdateGameStateJSON;
+    const { dealer, wind } = data;
+    const mjGameState = gameState as MahjongGameState;
+    mjGameState.gameStateSync(dealer, wind);
+    mjGameState.resetEverything();
+  };
+
+  /**
+   * For DRAW_ROUND
+   * @param payload ???????
```

Comment:
I set the payload to be:
```ts
export interface DrawRoundPayload {
  gameId: string;
  connectionId: string;
}
```
in the backend. Not quite sure if it needs anything in the payload, so I included something that might be useful to the client. Let me know if I need to add anything else to this payload

Reply:
ConnectionId, being the person who can't draw? I'm not sure what to put here as well. I'll put in those 2 as placeholders for now.

Reply:
ConnectionId is the user who failed to draw a tile, so the user before it got the last tile from the wall.